### PR TITLE
debug mode to bypass need for a handle server

### PR DESCRIPTION
**Pull request by qadan**

### CODE REVIEW

File: islandora_handle.install [hunk 1]
```diff
@@ -54,3 +54,21 @@ function islandora_handle_schema() {
   );
   return $schema;
 }
+
+/**
+ * Implements hook_uninstall()
+ */
+function islandora_handle_uninstall() {
```
Comment (willtp87) [inline islandora_handle.install]: An uninstall already exists.

File: includes/handle.inc [hunk 2]
```diff
@@ -27,13 +27,26 @@ function islandora_handle_construct_handle(AbstractObject $object) {
...
+  if (variable_get('islandora_handle_debug_mode', FALSE)) {
+    $response = new stdClass();
+    $response->code = 201;
+    db_insert('islandora_handle_debug_handles')
+      ->fields(array(
+        'suffix' => $suffix,
+        'handle' => $handle_service_url,
+      ))
+      ->execute();
+    return $response;
```
Comment (willtp87) [inline includes/handle.inc:40]: With the else the return isn't necessary and visa-versa; just one would be better.

File: includes/admin.inc [hunk 3]
```diff
@@ -79,6 +85,35 @@ function islandora_handle_server_form_submit($form, &$form_state) {
   variable_set('islandora_handle_server_admin_username', $form_state['values']['server_handle_admin_user']);
   variable_set('islandora_handle_server_admin_password', $form_state['values']['server_handle_admin_password']);
   variable_set('islandora_handle_server_prefix', $form_state['values']['server_handle_prefix']);
+  variable_set('islandora_handle_debug_mode', $form_state['values']['debug_mode']);
```
Comment (willtp87) [inline includes/admin.inc]: Should have consistency between 1,0 and true,false.

File: includes/admin.inc [hunk 4]
```diff
@@ -79,6 +85,35 @@ function islandora_handle_server_form_submit($form, &$form_state) {
...
+      'description' => 'Stores handles used during debug mode.',
+      'fields' => array(
+        'suffix' => array(
+          'description' => 'The suffix of the handle.',
+          'type' => 'varchar',
+          'length' => 255,
+          'not null' => TRUE,
+        ),
+        'handle' => array(
+          'description' => 'The handle for this suffix.',
```
Comment (willtp87) [inline includes/admin.inc]: This could be more clear.

File: islandora_handle.install (hunk 1 above)
Comment (qadan) [inline islandora_handle.install]: HOW DID I NOT NOTICE THIS

**Pull request merged**